authorization-service/serverless.ts

```diff
@@ -0,0 +1,75 @@
+const serverlessConfiguration = {
+    service: 'authorization-service',
+    useDotenv: true,
+    provider: {
+      name: 'aws',
+      runtime: 'nodejs20.x',
+      stage: 'dev',
+      region: 'eu-west-1',
+      apiGateway: {
+        minimumCompressionSize: 1024,
+        shouldStartNameWithService: true,
+      },
+      environment: {
+        REGION: 'eu-west-1',
+      },
+      iamRoleStatements: [
+        {
+          Effect: "Allow",
+          Action:  "execute-api:Invoke",
+          Resource: "*"
+        },
+      ],
+    },
+    resources: {
+      Resources: {
+        GatewayResponseAccessDenied: {
+          Type: 'AWS::ApiGateway::GatewayResponse',
+          Properties: {
+            ResponseParameters: {
+              'gatewayresponse.header.Access-Control-Allow-Origin': "'*'",
+              'gatewayresponse.header.Access-Control-Allow-Headers':
+                "'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token'",
+            },
+            ResponseType: 'ACCESS_DENIED',
+            RestApiId: 'cruvka2bu8',
```

Comment:
It's autogenerated id. If you remove the CloudFormation stack and recreate it, this id would be different. Lets try to build solution without that `RestApiId` reference

Reply:
done